docs/howto/white-label.md

```diff
@@ -0,0 +1,18 @@
+# White label an existing CommCareHQ instance
+
+Describes how to have two URLs point to the same CommCareHQ environment and
+serve separate branding for each.
+
+## Customization possible
+
+* Login page can be customized per host with default: `CUSTOM_LANDING_TEMPLATE` in public.yml 
+* CommCare HQ name can be customized per host with default: `COMMCARE_HQ_NAME` in public.yml 
+* CommCare name can be customized per host with default: `COMMCARE_NAME` in public.yml 
+* Error pages can be customized by creating a new branch in our `commcarehq_errors_repository`.
```

Comment:
Curious, and clearly missing something :upside_down_face: where is this repository?

Reply:
That's a variable defined in ansible, so its override-able. Hopefully made that a bit more clear in https://github.com/dimagi/commcare-cloud/pull/2580/commits/40401ddff085765b68fce407d430af72d6134cfb